test/unit/widgets/filter-input-test.js

```diff
@@ -20,7 +20,7 @@ describe("widgets > <FilterInput/>", () => {
     const wrapper = mountWithL10n(<FilterInput value="some text"/>);
     expect(wrapper.find("input")).to.have.prop("value", "some text");
     expect(wrapper.childAt(0).prop("className")).to.match(
-      /^\S+input-wrapper\S+$/
+      /^input.*input-wrapper\S+$/
```

Comment:
This should be `/^\S+filter\S+ \S+input-wrapper\S+$/` (the general rule is "put the un-munged classname between `\S+` on either side). The other diffs below should be adapted similarly.

Reply:
>the general rule is "put the un-munged classname between `\S+` on either side

Aha, thanks for the explainer.